ooiui/static/js/views/common/TroubleTicketView.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+/*
+* ooiui/static/js/views/common/TroubleTicketView.js
+*
+* Dependencies
+* Partials
+* - ooiui/static/js/partials/TroubleTicket.html
+*/
+var TroubleTicketView = Backbone.View.extend({
+  initialize: function() {
+    _.bindAll(this, "render");
+    console.log("binded");
+    this.render();
+  },
```

Comment:
@DanielJMaher You'll need a submit method that get's the form field data, puts it in the model and then saves the model.